gdal/ogr/ogrsf_frmts/mssqlspatial/ogrmssqlgeometryparser.cpp

```diff
@@ -31,7 +31,7 @@
 
 CPL_CVSID("$Id$")
 
-/*   SqlGeometry serialization format
+/*   SqlGeometry/SqlGeography serialization format
```

Comment:
this description is also repeated in geometrywriter.cpp. Perhaps move it in a mssqlgeometryformat.txt file ?

Reply:
Yes, but it was already there, however. I've just added the MSSQL2012 additions.